webapp/platform/components/src/generic_modal/generic_modal.tsx

```diff
@@ -128,7 +128,7 @@ export class GenericModal extends React.PureComponent<Props, State> {
                 <button
                     autoFocus={this.props.autoFocusConfirmButton}
                     type='submit'
-                    className={classNames('GenericModal__button', isConfirmOrDeleteClassName, this.props.confirmButtonClassName, {
```

Comment:
Also, if we are removing the `GenericModal__button` class, it seems to be used in several scss files (`category_modal.scss`, `product_notices_modal.scss` and `_post.scss`), and in `product_notices_modal.tsx`, so it would be interesting to verify that the usecase for all those places is still working as expected.